src/bobbit/message.py

```diff
@@ -13,12 +13,18 @@ def __init__(self, body, nick=None, channel=None, notice=False, highlighted=Fals
         self.timestamp   = timestamp or time.time()
 
     def copy(self, body=None, nick=None, channel=None, notice=None, highlighted=None, timestamp=None):
+        if notice is None:
+            notice = self.notice
+
+        if highlighted is None:
+            highlighted = self.highlighted
+
         return Message(
             body        = body        or self.body,
             nick        = nick        or self.nick,
             channel     = channel     or self.channel,
-            notice      = notice      or self.notice,
-            highlighted = highlighted or self.highlighted,
+            notice      = notice_truthy,
+            highlighted = highlighted_truthy,
```

Comment:
These also need to be updated.

We could also remove the `if` statements above by using Python's ternary statement as follows:

```
notice = self.notice if notice is None else notice
```

What do you think of that?  I'm OK with keeping your current structure if you think it is more readable.

Reply:
I'm fine with the ternary expression...I wasn't sure if you would think it too verbose for a single line. I'll change it to that